docs/deploying.md

```diff
@@ -2,155 +2,135 @@
 +++
 title = "Deploying a registry server"
 description = "Explains how to deploy a registry server"
-keywords = ["registry, service, images,  repository"]
+keywords = ["registry, service, images, repository"]
 [menu.main]
 parent="smn_registry"
 weight=3
 +++
 <![end-metadata]-->
 
-
 # Deploying a registry server
 
-You obviously need to [install Docker](https://docs.docker.com/installation/) (remember you need **Docker version 1.6.0 or newer**).
+You need to [install Docker version 1.6.0 or newer](https://docs.docker.com/installation/).
 
-## Getting started
+## Running on localhost
 
 Start your registry:
 
-     $ docker run -d -p 5000:5000 \
-     		--restart=always --name registry registry:2
+    docker run -d -p 5000:5000 --restart=always --name registry registry:2
 
-That's it.
+You can now use it with docker.
 
-You can now tag an image and push it:
+Get any image from the hub and tag it to point to your registry:
 
-    $ docker pull ubuntu && docker tag ubuntu localhost:5000/batman/ubuntu
-    $ docker push localhost:5000/batman/ubuntu
+    docker pull ubuntu && docker tag ubuntu localhost:5000/ubuntu
 
-Then pull it back:
+... then push it to your registry:
 
-    $ docker pull localhost:5000/batman/ubuntu
+    docker push localhost:5000/ubuntu
 
-## Where is my data?
+... then pull it back from your registry:
 
-By default, your registry stores its data on the local filesystem, inside the container.
+    docker pull localhost:5000/ubuntu
 
-In a production environment, it's highly recommended to use [another storage backend](storagedrivers.md), by [configuring it](configuration.md).
+Now, stop your registry:
```

Comment:
This makes it sound like stopping the registry is part of the process of running on localhost. Maybe say "To stop the registry:" instead?